book/website/collaboration/ppie-events/ppie-events-resources.md

```diff
@@ -0,0 +1,109 @@
+(cl-ppie-events-resources)=
+# Resources
+
+
+(cl-ppie-events-resources-learn)=
+## What to Learn Next
+
+> **Recommend other chapters in _The Turing Way_ book that readers should read next as a "level up", and explain why.**
```

Comment:
I think this might be a left-over from the template/guide, and could be removed? 🙂

Reply:
Yup! I hadn't noticed the second part of the sentence. Removing now.